src/software/ai/hl/stp/tactic/goalie/goalie_fsm.h

```diff
@@ -26,31 +27,22 @@ struct GoalieFSM
         bool should_move_to_goal_line;
     };
 
-    DEFINE_TACTIC_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
+    using Update = TacticFSM<GoalieFSM>::Update;
 
     // Distance to chip the ball when trying to yeet it
     // TODO (#1878): Replace this with a more intelligent chip distance system
     static constexpr double YEET_CHIP_DISTANCE_METERS = 2.0;
 
+
     /**
-     * Constructor for GoalieFSM struct
+     *  Constructor for GoalieFSM struct
```

Comment:
Nit: random whitespace in the docstring
```suggestion
     * Constructor for GoalieFSM struct
```

Reply:
How is it even possible to see these nits